release_tester/selenium_ui_test/pages/views_page.py

```diff
@@ -354,6 +337,152 @@ def create_improved_views(self, view_name, types):
+        views_sitem.click()
+        time.sleep(2)
+        self.wait_for_ajax()
+
+        print('Selecting Consolidation Policy \n')
+        policy = "//*[text()='Consolidation Policy']"
+        policy_sitem = self.locator_finder_by_xpath(policy)
+        policy_sitem.click()
+        time.sleep(2)
+
+        self.wait_for_ajax()
+        print('Selecting segments min value \n')
+        segment_min = "/html/body/div[2]/div/div[2]/div[2]/div/div/div[1]/div/div/table/tbody/tr[2]/th[2]/input"
+        segment_min_sitem = self.locator_finder_by_xpath(segment_min)
+        segment_min_sitem.click()
+        segment_min_sitem.clear()
+        segment_min_sitem.send_keys('4')
+        time.sleep(2)
+        self.wait_for_ajax()
+
+        print('Selecting segments max value \n')
+        segment_max = "/html/body/div[2]/div/div[2]/div[2]/div/div/div[1]/div/div/table/tbody/tr[3]/th[2]/input"
+        segment_max_sitem = self.locator_finder_by_xpath(segment_max)
+        segment_max_sitem.click()
+        segment_max_sitem.clear()
+        segment_max_sitem.send_keys('14')
+        time.sleep(2)
+
+        self.wait_for_ajax()
+        print('Selecting bytes value \n')
+        segment_bytes = "/html/body/div[2]/div/div[2]/div[2]/div/div/div[1]/div/div/table/tbody/tr[4]/th[2]/input"
+        segment_bytes_sitem = self.locator_finder_by_xpath(segment_bytes)
+        segment_bytes_sitem.click()
+        segment_bytes_sitem.clear()
+        segment_bytes_sitem.send_keys('5368709128')
+        time.sleep(2)
+
+        self.wait_for_ajax()
+        print('Selecting bytes floor value \n')
+        segment_floor = "/html/body/div[2]/div/div[2]/div[2]/div/div/div[1]/div/div/table/tbody/tr[5]/th[2]/input"
+        segment_floor_sitem = self.locator_finder_by_xpath(segment_floor)
+        segment_floor_sitem.click()
+        segment_floor_sitem.clear()
+        segment_floor_sitem.send_keys('2097158')
+        time.sleep(2)
+
+        print('Saving the consolidation policy with new value \n')
+        save = '//*[@id="modal-dialog"]/div[2]/button/i'
+        save_sitem = self.locator_finder_by_xpath(save)
+        save_sitem.click()
+        time.sleep(3)
+
+        self.wait_for_ajax()
+        print('Select JSON tab \n')
+        json = '//*[@id="subNavigationBar"]/ul[2]/li[5]/a'
+        json_sitem = self.locator_finder_by_xpath(json)
+        json_sitem.click()
+        time.sleep(1)
+
+        self.wait_for_ajax()
+        print("Switch editor mode to Compact mode Code \n")
+        compact = '//*[@id="JSON"]/div/div[2]/div/div/div/div/div[1]/button[2]'
+        compact_sitem = self.locator_finder_by_xpath(compact)
+        compact_sitem.click()
+        time.sleep(1)
+
+        print("Switch editor mode to normal mode Code \n")
+        normal = '//*[@id="JSON"]/div/div[2]/div/div/div/div/div[1]/button[1]'
+        normal_sitem = self.locator_finder_by_xpath(normal)
+        normal_sitem.click()
+
+        if deployment:
```

Comment:
if `deployment` contains the boolean state whether its a cluster, name it `is_cluster` pls.